docs/en/engines/table-engines/integrations/embedded-rocksdb.md

```diff
@@ -120,3 +120,88 @@ Values can be updated using the `ALTER TABLE` query. The primary key cannot be u
 ```sql
 ALTER TABLE test UPDATE v1 = v1 * 10 + 2 WHERE key LIKE 'some%' AND v3 > 3.1;
 ```
+
+### Joins
+
+A special `direct` join with EmbeddedRocksDB tables is supported.
+This direct join avoids forming a hash table in memory and accesses
+the data directly from the EmbeddedRocksDB.
+
+To enable direct joins:
+```sql
+SET join_algorithm = 'direct'
+```
+
+:::tip
+When the `join_algorithm` is set to `direct`, direct joins will be used
+when possible.  However, direct joins are not used for RIGHT or FULL JOINs.
+ClickHouse will choose another join algorithm when direct joins are not possible.
```

Comment:
Thanks very much; I will also add a link to https://clickhouse.com/docs/en/operations/settings/settings#settings-join_algorithm at the bottom of the page.

Reply:
Does that mean that in the tip I can change it to :
```
:::tip
When the `join_algorithm` is set to `direct, hash`, direct joins will be used
when possible, and hash otherwise.
:::
```